large_media

```diff
@@ -1 +1 @@
-Subproject commit 8e8b43ed3fe0649b5134313a2bdb3e36a3398075
+Subproject commit 3367112159d429a5ff70cf3c1466445f777d5015
```

Comment:
**Caution!** This contains a submodule update

Reply:
I don't think this was intentional, was it?

Reply:
@bwspenc This was intentional. The images I got from Wen for the xfem landing page I pushed to large_media at Andrew Slaughter's request. No one has updated the submodule since that repo has been updated, so this PR does that.

Reply:
OK, I figured that large_media was just updated periodically.  It looks like it's conflicting now, though, so maybe someone else did that. Once you resolve that, I'll merge this.